src/cljs/rems/actions/approve_reject.cljs

```diff
@@ -18,33 +16,21 @@
 (rf/reg-event-fx
  ::send-approve
  (fn [_ [_ {:keys [application-id comment on-finished]}]]
-   (let [description [text :t.actions/approve]]
-     (post! "/api/applications/approve"
-            {:params {:application-id application-id
-                      :comment comment}
-             :handler (flash-message/default-success-handler
-                       :actions
-                       description
-                       (fn [_]
-                         (collapse-action-form action-form-id)
-                         (on-finished)))
-             :error-handler (flash-message/default-error-handler :actions description)}))
+   (command! "approve" {:application-id application-id
```

Comment:
should the command name be a keyword? should we reuse the command names like `:application.command/approve`?

Reply:
We could also have it under a `:type` key, the same way as in the backend.

Reply:
Reusing the command names would make it easier to find every place where the command is used.